topdown/trace_test.go

```diff
@@ -576,7 +576,7 @@ query:3     | | Redo data.test.q[x]
 `
 
 	var buf bytes.Buffer
-	PrettyTraceWithLocation(&buf, *tracer)
+	PrettyTraceWithLocation(&buf, removeUnifyOps(*tracer))
```

Comment:
I think removing these in all except one test case makes sense since it's easier to update the test cases, they will need less frequent updates and it also keep them concise.  Let me know if you prefer differently.